### PR TITLE
Feature/Support for Swift 5

### PR DESCRIPTION
**Pull request by danielrobleM**
Support for Swift 5.0

### CODE REVIEW

File: Example/Pods/Local Podspecs/Heyou.podspec.json [hunk 1]
```diff
@@ -14,10 +14,11 @@
...
     "git": "https://github.com/etoledom/Heyou.git",
-    "tag": "0.2.3"
+    "tag": "0.3.0"
   },
   "platforms": {
     "ios": "10.0"
   },
-  "source_files": "Heyou/Classes/**/*"
+  "source_files": "Heyou/Classes/**/*",
+  "swift_version": "4.2"
```
Comment (etoledom) [inline Example/Pods/Local Podspecs/Heyou.podspec.json]: We probably want this one to be `"swift_version": "4.2"`.
It fixes for me locally after running `pod install`, could you please try it again locally too and let me know if you have some diff here?

**Review by etoledom — COMMENTED** [review]
Thank you for this update @danielrobleM ! It's looking great 🎉 

One thing I noticed is that after running `pod install` (cocoapods v1.8.4) I get some diffs.
In particular this one calls my attention:

```
--- a/Example/Pods/Local Podspecs/Heyou.podspec.json    
+++ b/Example/Pods/Local Podspecs/Heyou.podspec.json    
@@ -1,8 +1,8 @@
 {
   "name": "Heyou",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "summary": "Alerts",
-  "swift_versions": "4.2",
+  "swift_versions": "5.0",
   "description": "Alerts library to replace UIAlertController",
   "homepage": "https://github.com/etoledom/Heyou",
   "license": {
@@ -14,11 +14,11 @@
   },
   "source": {
     "git": "https://github.com/etoledom/Heyou.git",
-    "tag": "0.3.0"
+    "tag": "0.4.0"
   },
   "platforms": {
     "ios": "10.0"
   },
   "source_files": "Heyou/Classes/**/*",
-  "swift_version": "4.2"
+  "swift_version": "5.0"
 }
```

Also added a code comment relevant to this.

File: Example/Pods/Local Podspecs/Heyou.podspec.json (hunk 1 above)
Comment (danielrobleM) [reply Example/Pods/Local Podspecs/Heyou.podspec.json]: Hi @etoledom  , I set **5.0** as default to not have to manually change it, using something like 
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
       if ['Heyou'].include? target.name
          config.build_settings['SWIFT_VERSION'] = '5.0'
        end
    end
  end
end 
```
----------------------
Not sure why you see changes on `  "source_files": "Heyou/Classes/**/*"`

Screen Shot 2019-12-17 at 14 57 30

**Review by etoledom — APPROVED** [review]
Thank you for the update @danielrobleM !
Looking great now 🎉 

:shipit:

**Pull request merged**